### DISCUSSION TITLE
Understanding tailwind.config merge

### DISCUSSION THREAD

**Discussion by itpropro** [windicss/windicss#70] [Q&A] [answered]
Hi,
I just migrated a bigger Nuxt project from tailwind to windi and I have some issues. I think it's related to how the tailwind.config nodes are merged.
E.g. the colors plugin does not work correctly. I have a lot of cyan colors, which just don't get rendered, all non custom colors work. 
For testing, I removed everything from the tailwind.config and added 
```
const colors = require('windicss/colors')
module.exports = {
  theme: {
    extend: {
      colors: {
        cyan: colors.cyan,
      }
    }
  }
}
```
Then after I changed that to this and saw that several more styles broke:
```
const colors = require('windicss/colors')
module.exports = {
  theme: {
    colors: {
      cyan: colors.cyan,
    }
    extend: {
    }
  }
}
```
  I wonder what the actual merge strategy is and why there is a difference between adding it to the `extend` node compared to the `theme` node.

**Comment by hannoeru** [comment, marked as answer]
Just fixed it and will be release on next version,
the behavior should be the same as tailwindcss now,
theme node will replace default theme, and extends will extend default theme.
